### PR TITLE
First samples for model catalog

### PR DESCRIPTION
**Pull request by hchandola**
# Description

Adds samples for first set of models for model catalog. 

# Checklist

- [ ] I have read the [contribution guidelines](https://github.com/Azure-Samples/azureai-samples/blob/main/CONTRIBUTING.md)
- [ ] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [ ] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure-Samples/azureai-samples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

File: model-catalog/samples/mistral-large/curl/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+## 1. Create a personal access token
+
+You'll need this to enable free API access. The token will be sent to a Microsoft service but does not need any permissions.
+
+## 3. Use your personal access token and get model endpooint
+
+Update or create an environment variable to set your token as the key for the client code.
+
+```bash
+export GITHUB_MODELS_API_KEY="<your-key-goes-here>"
```
Comment (jfomhover) [inline model-catalog/samples/mistral-large/curl/getting_started.md:17]: nit: can we put endpoint first?

**Review by jfomhover — APPROVED** [review]
it looks great for a first batch, let's merge as-is and iterate

**Review by diondrapeck — CHANGES_REQUESTED** [review]
Hi Himanshu - thank you for making a contribution to the repository! Please see the contributing guidelines [here](https://github.com/Azure-Samples/azureai-samples/blob/main/CONTRIBUTING.md#write-your-contribution), especially the section around adding READMEs to make your PR compliant with our standards.

**Comment by qubitron** [comment]
@diondrapeck talked with the team, we're going to auto-generate the readmes but don't have code for that yet, we'll follow up with another PR for that but need to urgently merge this in

**Pull request merged**